nss-day-cohort-52/giffygram-retro-blasters
# Delete Button Issue #7

#### Changes Made

1. Modified file `PostList` to include a delete button
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sr-delete
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Rendering a delete button below the post - needs code to work/use trashcan icon found with block.svg 


4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

*confirming button is shown but does not execute*
